### PR TITLE
:sparkles: Add ALM Examples for new CRs

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by jmontleon** [comment]
This PR is dependent on and should be rebased once https://github.com/konveyor/operator/pull/368 is merged.

**Comment by jmontleon** [comment]
@jortel can you please have a quick look and make sure these examples aren't insane.

**Comment by jmontleon** [comment]
@fbladilo heads up on this one too. In downstream won't take the ALM example templates into account and you need to manually update the CSV (or ignore this/me if the opposite is true.)

File: helm/templates/crds/tackle.konveyor.io_addons.sample.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    imagePullPolicy: Always
+    name: myaddon
+    resources:
+      limits:
+        cpu: 1
+        memory: 1Gi
+      requests:
+        cpu: 1
+        memory: 512Mi
+  resources:
```
Comment (jortel) [inline helm/templates/crds/tackle.konveyor.io_addons.sample.yaml:19]: I don't think we want this `resources`.

File: bundle/manifests/konveyor-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -5,7 +5,86 @@ metadata:
...
+                  "cpu": 1,
+                  "memory": "1Gi"
+                },
+                "requests": {
+                  "cpu": 1,
+                  "memory": "512Mi"
+                }
+              }
+            },
+            "resources": {
```
Comment (jortel) [inline bundle/manifests/konveyor-operator.clusterserviceversion.yaml:30]: I don't think we want this `resources` on line 30.

Comment (jmontleon) [reply bundle/manifests/konveyor-operator.clusterserviceversion.yaml]: Thank you, updated!

File: helm/templates/crds/tackle.konveyor.io_addons.sample.yaml (hunk 1 above)
Comment (jmontleon) [reply helm/templates/crds/tackle.konveyor.io_addons.sample.yaml]: Thank you, updated!

**Review by jortel — APPROVED** [review]
lgtm

**Pull request merged**